plugins/jobtype/src/azkaban/jobtype/connectors/HdfsToTeradataJobRunnerMain.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.jobtype.connectors;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.security.PrivilegedExceptionAction;
+import java.util.Properties;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PatternLayout;
+
+import azkaban.jobExecutor.ProcessJob;
+import azkaban.jobtype.*;
+import azkaban.jobtype.javautils.JobUtils;
+import azkaban.security.commons.HadoopSecurityManager;
+import azkaban.utils.Props;
+
+import com.teradata.hadoop.tool.TeradataExportTool;
+
+import static org.apache.hadoop.security.UserGroupInformation.HADOOP_TOKEN_FILE_LOCATION;
+
+public class HdfsToTeradataJobRunnerMain {
+  private static final int DEFAULT_NO_MAPPERS = 8;
+
+  private final Properties _jobProps;
+  private final TdchParameters _params;
+  private final Logger _logger;
+
+  public HdfsToTeradataJobRunnerMain() throws FileNotFoundException, IOException {
+    _logger = JobUtils.initJobLogger();
+    _jobProps = HadoopSecureWrapperUtils.loadAzkabanProps();
+    _jobProps.put(HadoopSecurityManager.ENABLE_PROXYING, "true"); //Always headless account
+
+    Props props = new Props(null, _jobProps);
+    HadoopConfigurationInjector.injectResources(props);
+    UserGroupInformation.setConfiguration(new Configuration());
+    props.getString(HadoopSecurityManager.USER_TO_PROXY); //Check required field.
```

Comment:
Something seems off. We are not consuming returned value of props.getString(HadoopSecurityManager.USER_TO_PROXY) ?